app/src/main/res/values/dimens.xml

```diff
@@ -3,5 +3,7 @@
     <dimen name="toolbar_shadow_size">1dp</dimen>
     <dimen name="primary_button_radius">16dp</dimen>
     <dimen name="safe_blockie_size">36dp</dimen>
+    <dimen name="default_horizontal_margin">16dp</dimen>
+    <dimen name="default_line_spacing">6sp</dimen>
```

Comment:
Spacing should be probably in `dp`. I would also ask you that even if you see a specific value that is not a multiple of 8, you truncate it to the closest one, we otherwise can't ensure correct alignment of views in layouts.

Reply:
I took this value from Sashas Layout. IIRC we had this in th stayles already. MAybe this is sp because it refers to fonts?